index.js

```diff
@@ -17,4 +17,25 @@ export default {
     },
 
     generateTypeScript
-}
\ No newline at end of file
+}
+
+export { ControlTouchPadInputEventData } from './src/platform/lumin-runtime/types/event-data/control-touch-pad-input-event-data.js';
```

Comment:
What about iOS/Android implementations.  What happens if an app uses these and tries to compile for react-native?

Reply:
I don't have clear solution for this case as of now since some of those events are pure Lumin Runtime. That is why the developer should import from `magic-script-components-lumin` thus clearly making this code running on ML1 only. At some point we will add support for most of those events for iOS and Android. However, until this happens I would like the ML1 developers to be able to use them.

Reply:
We don't want developers to have to import from Lumin directly. This will break iOS/Android apps. We can expose the events to the top level components and document that for React Native they won't get these, only on Lumin. Then as they're implemented we update our docs.